xbmc/interfaces/json-rpc/FavouritesOperations.cpp

```diff
@@ -53,7 +54,12 @@ JSONRPC_STATUS CFavouritesOperations::GetFavourites(const std::string &method, I
 
     std::string function;
     std::vector<std::string> parameters;
-    CUtil::SplitExecFunction(item->GetPath(), function, parameters);
+
+    //FIXME: this path is internal to the favourites system and should not be parsed and exposed
```

Comment:
Not sure what point this comment serves hidden here. If anything then add a Doxygen "todo" tag